### DISCUSSION TITLE
Empyrion: Galactic Survival

### DISCUSSION THREAD

**Discussion by mvaessen** [gameserverapp/Platform#32] [Game request]
https://empyrion.fandom.com/wiki/Dedicated_Server_Setup

**Comment by Deluezeonest** [comment]
Yes! This game is amazing. Its basically three genres in one. Got the survival aspects of ark, the design and building aspects of avorion and then the awesome PVE & PVP styled combat of Elite & Dangerous. Love this game.

**Comment by Swayz3** [comment]
https://empyrion.fandom.com/wiki/Telnet_Commands

**Comment by zarrstlouis** [comment]
@mvaessen  How are we looking at getting Empyrion added to GSA?

**Reply by mvaessen** [reply]
Hi, I'm not able to provide any eta's. But soon you'll be able to add custom games. So theoretically that would allow you to add any game to the platform.

**Comment by mvaessen** [comment]
This game is now available on the dashboard.
